decentraland/unity-renderer
# feat: [BiW Editor] Create a new Feedback Tooltip View

## What this PR includes?

### Fixes #474 (**BiW editor disabled Publish button should give feedback**)
A tooltip with a message should appear displaying this information:
- [x] Scene limits reached

![image](https://user-images.githubusercontent.com/64659061/119829440-873e2e00-befb-11eb-9e7a-00d20c3e1fac.png)

- [x] There are entities outside of the scene boundaries.

![image](https://user-images.githubusercontent.com/64659061/119829168-40503880-befb-11eb-92fd-7a2f401ebe25.png)

## How to test it?

1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:feat/BiW-disabled-Publish-button-should-give-feedback&ENV=zone&ENABLE_BUILDER_IN_WORLD&position=80,154
2. Press 'K' to open the Builer In World editor.

## Our Code Review Standards
https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
